<ui version="4.0" >
 <class>ListarPresupuestosClass</class>
 <widget class="QWidget" name="ListarPresupuestosClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ListarPresupuestos</string>
  </property>
  <property name="windowIcon" >
   <iconset>images/LPresupuesto.PNG</iconset>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTableWidget" name="tableWidget" >
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text" >
        <string>NÃºmero</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Fecha</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Apellidos</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Aparato</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Marca</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pushButton_2" >
        <property name="text" >
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3" >
        <property name="text" >
         <string>Borrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ListarPresupuestosClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>326</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel" >
     <x>332</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
